docs/topics/http/shortcuts.txt

```diff
@@ -107,8 +107,28 @@ This example is equivalent to::
    * An absolute or relative URL, which will be used as-is for the redirect
      location.
 
-   By default issues a temporary redirect; pass ``permanent=True`` to issue a
-   permanent redirect.
+   By default, this issues a temporary redirect (status code 302).
+
+   If ``permanent=True``, this issues a permanent redirect (status code 301).
+
+   If ``preserve_request_method=True``, the method and body of the original
+   request are reused to perform the redirected request. Temporary redirects
+   then have a 307 status code and permanent redirects have a 308 status code.
```

Comment:
This reads a bit odd in that Django does not do anything, the one that uses a status code to issue a new request to the redirected Location is the caller (user agent). This just tells in the response "you should go to this other place and tell them the same things you told me".

Proposal:
```suggestion
   If ``preserve_request_method=True``, the redirect instructs the user agent
   to reuse the method and body of the original request when performing the
   redirected request to the ``Location`` value. Temporary redirects then use a
    307 status code, and permanent redirects use a 308 status code.
```